doc/source/tutorial-series-get-started-with-flower-pytorch.ipynb

```diff
@@ -640,9 +620,21 @@
    "toc_visible": true
   },
   "kernelspec": {
-   "display_name": "flower-3.7.12",
+   "display_name": "flwr-rtwXnbAq-py3.10",
```

Comment:
@adam-narozniak just to be sure, is this intended?

Reply:
that change probably comes from my side when re-running the notebook. i think this is not important. Can be discarded or kept.

Reply:
@tanertopal both "display_name" and "name" are required. I changed it to "flwr" now.
When they are not present I get "[E 09:57:23.746 NotebookApp] Notebook JSON is invalid: 'display_name' is a required property". And in UI, when starting the notebook I get the error (if not present). Similarly when "name" was not present in google colab, it complained and defaulted to "python3"